labs/SizerBase/samples/SizerBase.Samples/GridSplitter.md

```diff
@@ -0,0 +1,18 @@
+---
+title: GridSplitter
+author: mhawker
+description: The GridSplitter control provides an easy-to-use Splitter that redistributes space between columns or rows of a Grid Control.
+keywords: ContentSizer, SizerBase, Control, Layout, Expander
+dev_langs:
+  - csharp
+category: Controls
+subcategory: Layout
+
```

Comment:
@LalithaNadimpalli you are missing the closing `---` for the front-matter section, so that's why there was an error. Just compare to one of the other MD docs.

Reply:
Created a new PR again. I am closing this just to avoid any further confusion